api/Node.json

```
@@ -1040,10 +1040,12 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": "≤4",
```

```suggestion
              "version_added": "1",
```

Based on https://trac.webkit.org/browser/webkit/trunk/WebCore/khtml/ecma/kjs_dom.lut.h?rev=4552